src/Style/Style.scss

```diff
@@ -218,6 +219,7 @@ div[id="content"]
   		width: 100%;
   		height: 100%;
 		mask-size: contain;
+		image-rendering: -webkit-optimize-contrast;
```

Comment:
What is this for? I'm not able to find any documentation on this property, which means it may be unsupported

Reply:
> What is this for? I'm not able to find any documentation on this property, which means it may be unsupported

With changes:
![image](https://user-images.githubusercontent.com/60571306/160202993-975e9662-fe8f-4b28-9de8-217d9309a870.png)
Prior to changes:
![image](https://user-images.githubusercontent.com/60571306/160203026-9e7cbf13-aaff-4334-8496-73e9c4c26f31.png)

In essence, optimized contrast is a rendering mode that gives "crisp edges"

The property works on: Chrome, Edge, Opera, and Safari. (https://www.w3schools.com/cssref/tryit.asp?filename=trycss3_image-rendering)

When on Mozilla firefox I believe the image rendering is automatically different as the logo does not appear to be blurry as seen below (without changed image rendering.)
Mozilla:
![image](https://user-images.githubusercontent.com/60571306/163434590-c162f71a-a9c5-4622-9cf5-2852eea37c5c.png)
Chrome:
![image](https://user-images.githubusercontent.com/60571306/163434625-f12ecdb0-6462-4e81-a524-b4bd491b1136.png)

Overall, the change is extremely minute and is not that integral. You could also probably make use of an SVG instead to prevent blurring.

Reply:
Can you remove this? The other change seems fine, but I don't like this change. The icon could be changed to a pre-scaled image or a vector, applying this property doesn't seem to make it look better; just more pixelated.

Reply:
Reverted 5986729d65b23277ede8d86e620681090f839f8e